openedx/features/badging/tabs.py

```diff
@@ -1,4 +1,9 @@
+"""
+This module is essentially a broker to `xmodule/tabs.py`, It was originally introduced
```

Comment:
This module is essentially a broker to `xmodule/tabs.py`**.** It was originally introduced